app/views/mappings/_mapping_table.html.haml

```diff
@@ -12,7 +12,7 @@
 = check_box_tag "delete_mappings_permission", @delete_mapping_permission, @delete_mapping_permission, style: "display: none;"
 %div#concept_mappings_tables_div
   = render_alerts_container(MappingsController)
-  %table#concept_mappings_table.zebra{width: "100%", style:'word-break: break-word'}
+  %table#concept_mappings_table.table-content-stripped.table-content{width: "100%", style:'word-break: break-word'}
```

Comment:
use the Table component here, no just using its style.

Reply:
There is a lot of javascript code to be rewriten this is why I prefered to use only the style, the day we migrate the js code we can use the new component

Reply:
OK